test/sql/alter/alter_column/test_drop_not_null.test

```diff
@@ -0,0 +1,45 @@
+# name: test/sql/alter/not_null/test_drop_not_null.test
+# description: Test ALTER TABLE ALTER COLUMN column_name DROP NOT NULL
+# group: [alter_column]
+
+# Happy path
+
+statement ok
+CREATE TABLE test(i INTEGER, j INTEGER NOT NULL)
```

Comment:
Could you add the following tests:

* Tests with a larger table
* Tests with adding constraints to tables with uncommitted changes
* Rolling back the adding and removal of constraints
* Testing of adding constraints and removing in the presence of multiple transactions
* Testing of adding and removing constraints with the storage and the WAL
* Adding and dropping constraints within the same transaction